helix-term/src/keymap.rs

```diff
@@ -493,10 +493,11 @@ impl Default for Keymaps {
             },
 
             "/" => search,
-            // ? for search_reverse
+            "?" => rsearch,
             "n" => search_next,
-            "N" => extend_search_next,
-            // N for search_prev
+            "N" => search_prev,
+            "L" => extend_search_next,
+            "H" => extend_search_prev,
```

Comment:
See how the keys are laid out here: https://github.com/helix-editor/helix/pull/211/files#diff-8b8f04b030b7fdea89f3d9ec2b7aeb6a33959ee4c6cb27f0d574f115b474d151R420

`N` becomes search previous, then `n`/`N` in select mode extend.